README.md

```diff
@@ -61,21 +62,59 @@ __Please be aware that although OpenMage is compatible that one or more extensio
 
 ## Installation
 
+### Manual Install
+
+Download the latest [release archive](https://github.com/OpenMage/magento-lts/releases) and extract it over your existing install. **Important:** you must download the ZIP file from a tagged version on the releases page, otherwise there will be missing dependencies.
```

Comment:
did we have this part before like this? because extracting it over an existing install is only updating/adding files. But files which got deleted or moved will stay in their place, which in some cases might be a security issue.

Reply:
@Flyingmana I think it's related to the "composer thing". It is true that just overwriting everything can leave orphaned files, maybe we could write it in a better way. It's still a difficult task since, if people installed everything in the same docroot, how will they be able to "remove everything" and reinstall? They'll for sure have css/js and other stuff in various directories that make everything more difficult.

BTW we have a specific section "Since OpenMage 19.5.0 / 20.1.0" in the readme with a "rm" command, check it out ;-)

Reply:
what we could do is updating the migration project https://github.com/OpenMage/migrate/blob/master/create-patches.sh to create diff based patches as in the old days. Also github offers patch files for release upgrades as for example https://github.com/OpenMage/magento-lts/compare/v20.0.17...v20.0.18.patch although thats not covering the composer dependencies, so the migration project which creates and hosts patches might be a valuable approach

Reply:
I suppose that this is the "classic" way to install / upgrade Magento. Not the best, but is for people who just use FTP.

I wouldn't personally work on creating diff based patches since simply using composer would be a better approach and we should be encouraging that way intead.